public/locales/es/AssignmentTexterContact.json

```
@@ -0,0 +1,14 @@
+{
+  "can'tSendEmptyMessage": "No puedes enviar un mensaje vacío.",
+  "sending": "Enviando...",
+  "assignmentChanged": "Esta asignación ha cambiado.",
+  "backToTodos": "Volver a pendientes",
```

Question: Are you familiar with the word pendientes used in this way? Google Translate says it means earrings 😅 If you haven't seen it before, I might prefer the word tareas which I think is pretty common and means tasks
```suggestion
  "backToTodos": "Volver a tareas",
```